### PR TITLE
add context to react relay

### PR DESCRIPTION
**Pull request by itajaja**
see https://github.com/facebook/relay/blob/master/packages/react-relay/index.js#L13

should I add tests?

### CODE REVIEW

File: types/react-relay/index.d.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@
 export {
     commitLocalUpdate,
     commitRelayModernMutation as commitMutation,
+    CRelayContext,
+    Environment,
```
Comment (alloy) [inline types/react-relay/index.d.ts]: These are exports, not imports, hence the CI failure.

**Review by alloy — CHANGES_REQUESTED** [review]
Thanks!

Yeah, a test would be great 👍

File: types/react-relay/index.d.ts (hunk 1 above)
Comment (itajaja) [reply types/react-relay/index.d.ts]: yes lol. I tried to cowboy my way from the github editor :P

**Comment by alloy** [comment]
Will you be adding a test?

**Comment by itajaja** [comment]
what do you think? is it needed? I have never done "typing" tests so your call

**Comment by alloy** [comment]
Yes, I would like new additions to have test coverage. A test is really just a snippet of legit usage of the types.

File: types/react-relay/test/react-relay-tests.tsx [hunk 2]
```diff
@@ -501,3 +502,11 @@ requestSubscription(
         },
     }
 );
+
+
+// ~~~~~~~~~~~~~~~~~~~~~
+// Context
+// ~~~~~~~~~~~~~~~~~~~~~
+
+ReactRelayContext.Consumer.prototype
```
Comment (itajaja) [inline types/react-relay/test/react-relay-tests.tsx]: making sure that ReactRelayContext and it has the expected properties

File: types/react-relay/test/react-relay-tests.tsx [hunk 3]
```diff
@@ -501,3 +502,10 @@ requestSubscription(
         },
     }
 );
+
+// ~~~~~~~~~~~~~~~~~~~~~
+// Context
+// ~~~~~~~~~~~~~~~~~~~~~
+
+ReactRelayContext.Consumer.prototype;
+ReactRelayContext.Provider.prototype;
```
Comment (alloy) [inline types/react-relay/test/react-relay-tests.tsx]: I actually meant a snippet of actual API usage, but at a glance I only see `createFragmentSpecResolver` as a potential option and it doesn’t seem trivial to use that right now, so let’s just go with this for now 👍

**Review by alloy — APPROVED** [review]

File: types/react-relay/test/react-relay-tests.tsx (hunk 3 above)
Comment (itajaja) [reply types/react-relay/test/react-relay-tests.tsx]: yeah. I mean, I didn't actually specify the Context type or any other type in this PR, that already exists and is hopefully tested in relay-runtime. i just added a `const` in the module, so this tests that the `const` is indeed there 😅

Comment (alloy) [reply types/react-relay/test/react-relay-tests.tsx]: Right right, I just try to identify opportunities for backfilling some of the debt we have as well ;)

**Pull request merged**